alerting/build.gradle

```diff
@@ -59,7 +59,13 @@ configurations.all {
         force "commons-logging:commons-logging:${versions.commonslogging}"
         force "org.apache.httpcomponents:httpcore:${versions.httpcore}"
         force "commons-codec:commons-codec:${versions.commonscodec}"
-        
+        // commons-validator 1.6 depends on commons-collections 3.2.1 which causes the conflict
+        force "commons-collections:commons-collections:3.2.2"
+        // httpasyncclient 4.1.4 depends on httpcore-nio 4.4.10 which causes the conflict
+        force "org.apache.httpcomponents:httpcore-nio:4.4.12"
+        // httpasyncclient 4.1.4 depends on httpclient 4.5.6 which causes the conflict
+        force "org.apache.httpcomponents:httpclient:${versions.httpclient}"
```

Comment:
I changed the version number of some dependencies and add some comments (Here is the [code](https://github.com/ann3431/alerting/blob/430d718f96803ceec00b5fb50e664300fde6fa21/alerting/build.gradle#L52) in the original PR)